### DISCUSSION TITLE
Welcome to the Dev Container Spec's Discussions!

### DISCUSSION THREAD

**Discussion by bamurtaugh** [devcontainers/spec#52] [Announcements]
## 👋 Welcome!
  Welcome to Discussions on the development container specification! We’re using Discussions as a place to openly discuss questions and feedback on the spec and connect with other members of our community. We hope that you:
  * Ask questions - these could be about spec content in this repo or in other repos in the [devcontainers org](https://github.com/devcontainers).
  * Share ideas, i.e. potential additions to the spec. 
       * Let us know how you or your team may use dev containers, and anything that'd make them more useful for you. We'd love to collaborate on your feedback and continue evolving this spec!
  * Engage with other community members and are open-minded.

  To get started, please feel free to open a Discussion or review other Discussions and issues in this repo. We can't wait to hear from you!

**Comment by Bunyuh** [comment]
> 
> ## 👋 Welcome!
>   Welcome to Discussions on the development container specification! We’re using Discussions as a place to openly discuss questions and feedback on the spec and connect with other members of our community. We hope that you:
>   * Ask questions - these could be about spec content in this repo or in other repos in the [devcontainers org](https://github.com/devcontainers).
>   * Share ideas, i.e. potential additions to the spec. 
>        * Let us know how you or your team may use dev containers, and anything that'd make them more useful for you. We'd love to collaborate on your feedback and continue evolving this spec!
>   * Engage with other community members and are open-minded.
> 
>   To get started, please feel free to open a Discussion or review other Discussions and issues in this repo. We can't wait to hear from you!
> 
> 
>

**Reply by ghost** [reply]
Ahoj ,především přeji šťastný nový rok a hlavně nejdůležitější Zdravííí.
Moc rád se podílím na spolupráci,ale zatím si tak plavu ,jestli vás mohu poprosit dejte mi vědět co by bylo užitečné Děkuji s přátelským pozdravem
